projects/ifa-dissertation/js/index.js

```diff
@@ -92,27 +90,28 @@ d3.csv("./data/ifa-dissertations.csv", function(error, data) {
       count: advCount[k]
     });
   });
-
-  // console.log(catData);
-
   categories = categories.unique();
   advisors = advisors.unique();
 
 
-  catData = catData.map(function(d){ d.value = +d.count; return d; });
+  catData = catData.map(function(d){ 
+    d.value = +d.count;
+    d.centerX = diameter/2;
+    d.centerY = diameter/2;
```

Comment:
adding `centerX` and `centerY` to each element of data `d`. This represents the coordinate to which the all the nodes flock to. In our case it's the center of the `svg` element.